### ISSUE TITLE
Modal: Dialog: Prompt: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by timrasche** [uikit/uikit#4777]
### UIkit Version
3.16.15

### Actual Behavior
Create modal with UIkit.modal.confirm with opened offcanvas produces uncountable errors 

### Expected Behavior
This should work as in documentation expected

**Comment by janschoenherr** [comment]
I need more details please.

**Comment by timrasche** [comment]
Okay strange. I'm using stimulus framework, custom function, that calls the prompt method.

`function preventBackgroundFocus(modal) {
      return on(document, "focusin", (e) => {
        if (last(active) === modal && !within(e.target, modal.$el)) {
          modal.$el.focus();
        }
      });
    }`

modal.$el.focus(); <--- this produces "caught RangeError: Maximum call stack size exceeded."

Otherwise i dont use any other special thing, plain symfony 6.

Hope that helps.

**Comment by timrasche** [comment]
Nope, there is a stimulus controller function that calls the method separated in a "function" js file that executes internally UIkit.modal.confirm();

**Comment by janschoenherr** [comment]
And at what point? Is the Offcanvas opened completely before? Is there some kind of user interaction before?

**Comment by timrasche** [comment]
1. Open offcanvas
2. Async content will be loaded and injected
3. Some action called (button click, select change etc.)
4. The action is defined in the stimulus controller
5. The action calls a method in an imported javascript file with function declaration
6. The function calls UIkit.modal.confirm and produces immediatly the error described before

**Comment by janschoenherr** [comment]
Sorry, I tried my best. I can't reproduce it.
